src/OrchardCore/OrchardCore.DisplayManagement/TagHelpers/BaseShapeTagHelper.cs

```diff
@@ -108,6 +111,8 @@ protected BaseShapeTagHelper(IShapeFactory shapeFactory, IDisplayHelperFactory d
                 }
             }
 
+            await output.GetChildContentAsync();
```

Comment:
This method is annotated with `MustUseReturnValue`, but I'm not sure why, or how I should be using it?

Additionally- `SetHtmlContent` called immediately after this line is also annotated with this attribute, but the return value is also being ignored here.